platforms/documentation/docs/src/snippets/bestPractices/useCacheabilityAnnotations/tests/useCacheabilityAnnotations.sample.conf

```diff
@@ -0,0 +1,37 @@
+commands: [{
+    executable: gradle
+    args: addBad1 addBad2
+    expected-output-file: useCacheabilityAnnotations.bad.out
+    allow-disordered-output: true
+    allow-additional-output: true
+    flags: --build-cache
+}, {
+     executable: gradle
+     args: clean
```

Comment:
💅  whitespace formatting is slightly off

Reply:
Good eye!  I'll correct this.